tests/org.eclipse.epsilon.emc.rdf.tests/src/org/eclipse/epsilon/emc/rdf/RDFModelValidationTest.java

```diff
@@ -0,0 +1,81 @@
+/********************************************************************************
+ * Copyright (c) 2024 University of York
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Antonio Garcia-Dominguez - initial API and implementation
+ ********************************************************************************/
+package org.eclipse.epsilon.emc.rdf;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Collection;
+
+import org.eclipse.epsilon.common.util.StringProperties;
+import org.eclipse.epsilon.eol.exceptions.models.EolModelLoadingException;
+import org.eclipse.epsilon.eol.execute.context.EolContext;
+import org.junit.After;
+import org.junit.Test;
+
+public class RDFModelValidationTest {
+
+	private static final String OWL_DEMO_DATAMODEL_INVALID = "resources/OWL/owlDemoData.ttl";
+	private static final String OWL_DEMO_DATAMODEL_VALID = "resources/OWL/owlDemoData_valid.ttl";
+	private static final String OWL_DEMO_SCHEMAMODEL = "resources/OWL/owlDemoSchema.ttl";
+
+	private static final String LANGUAGE_PREFERENCE_EN_STRING = "en";
+	private static final String URI_BIGNAME42 = "urn:x-hp:eg/bigName42";
+	private static final String URI_ALIENBOX51 = "urn:x-hp:eg/alienBox51";
+	private static final String URI_WHITEBOX = "urn:x-hp:eg/whiteBoxZX";
+
+	private RDFModel model;
+	private EolContext context;
+
+	@Test
+	public void loadInvalidModel() {
+
+		try {
+			loadModel(OWL_DEMO_DATAMODEL_INVALID);
+		} catch (EolModelLoadingException e) {
+			String sErrors = e.getMessage();
+			assertTrue("The model loaded should FAIL validation (6 errors relating to BigName42 having 2 motherboards)",
+					sErrors.contains("Error whilst loading model null: The loaded model is not valid or not clean"));
```

Comment:
```suggestion
					sErrors.contains("not valid"));
```

I'd suggest relaxing this a bit to just mentioning that the model is not valid. We don't want the test to be too brittle.

What does it mean for a model to be "not clean"?

Reply:
https://jena.apache.org/documentation/inference/index.html#validation

_"A particular case of this arises in the case of OWL. In the Description Logic community a class which cannot have an instance is regarded as "inconsistent". That term is used because it generally arises from an error in the ontology. However, it is not a logical inconsistency - i.e. something giving rise to a contradiction. Having an instance of such a class is, clearly a logical error. In the Jena 2.2 release we clarified the semantics of isValid(). An ontology which is logically consistent but contains empty classes is regarded as valid (that is isValid() is false only if there is a logical inconsistency). Class expressions which cannot be instantiated are treated as warnings rather than errors. **To make it easier to test for this case there is an additional method Report.isClean() which returns true if the ontology is both valid (logically consistent) and generated no warnings (such as inconsistent classes)**."_